app/controllers/actions/SubmittedDataRetrievalActionProvider.scala

```diff
@@ -76,7 +77,13 @@ class SubmittedDataRetrievalActionProviderImpl @Inject() (connector: SelfEmploym
         gtsouUpdated,
         Journey.CapitalAllowancesTailoring
       ) // TODO 10281 fetch Profit or Loss journey answers
```

Comment:
Not part of your ticket but could we fetch the Profit or Loss answers as well here since that has been missed previously